src/app/components/elements/layout/ColumnSlice.tsx

```diff
@@ -0,0 +1,14 @@
+import classNames from "classnames";
+import React from "react";
+import { Row, RowProps } from "reactstrap";
+
+interface ColumnSliceProps extends RowProps {
+    breakpoint?: "sm" | "md" | "lg" | "xl";
+}
+
+export const ColumnSlice = ({...props}: ColumnSliceProps) => {
+    const numChildren = Math.min(React.Children.count(props.children), 4);
+    return <Row {...props} className={classNames(`row-cols-1 row-cols-${props.breakpoint ?? "md"}-${numChildren}`, props.className)}>
```

Comment:
I think `lg` is a sensible default given the designs:

```suggestion
    return <Row {...props} className={classNames(`row-cols-1 row-cols-${props.breakpoint ?? "lg"}-${numChildren}`, props.className)}>
```